showcase/src/app/demo/keyboard-directives/keyboard-directives.component.ts

```diff
@@ -0,0 +1,48 @@
+import { Component, Inject, OnInit } from "@angular/core";
+import { STARK_LOGGING_SERVICE, StarkLoggingService } from "@nationalbankbelgium/stark-core";
+
+@Component({
+	selector: "showcase-demo-on-enter-key",
+	styleUrls: ["./keyboard-directives.component.scss"],
+	templateUrl: "./keyboard-directives.component.html"
+})
+export class KeyboardDirectivesComponent implements OnInit {
+	public latestInputValue: string;
+	public inputValue1: string;
+	public inputValue2: string;
+	public inputValue3: string;
+
+	public logging: string;
+
+	public constructor(@Inject(STARK_LOGGING_SERVICE) private logger: StarkLoggingService) {}
+
+	public ngOnInit(): void {
+		this.latestInputValue = "";
+		this.inputValue1 = "";
+		this.inputValue2 = "";
+		this.inputValue3 = "";
+		this.logging = "";
+	}
+
+	public onEnterKeyCallback(...paramValues: any[]): void {
+		if (typeof this.latestInputValue !== "undefined") {
+			this.latestInputValue = paramValues[1];
```

Comment:
I think you do a method 'onEnterKeyCallbackValueX' for each input example.
This method could obviously call a common method like `onEnterKeyCallback(latestInputValue, paramValues)`

And so for the latestInputValue use `this.inputValueX` instead of paramValues[1] which is actually not the latestInputValue the component got. 
We should be sure that ngModel is working....

Reply:
And latestInputValue could be changed by ngModelLatestValue or something like this 😊